index.js

```diff
@@ -45,8 +45,8 @@ module.exports.plugins = [
       { no: "Github", yes: "GitHub" },
       { no: "Javascript", yes: "JavaScript" },
       { no: "Node.JS", yes: "Node.js" },
-      { no: "Note that", yes: "<nothing>" },
-      { no: "note that", yes: "<nothing>" },
+      { no: "\\s+node.js", yes: "Node.js" },
+      { no: "[Nn]ote that", yes: "<nothing>" },
```

Comment:
I wonder if this should get a `\\b` out front so that it doesn't flag "Use 'x' to denote that something something."

Reply:
Should the `\\b` be part of the plugin? Do we envisage scenarios where the prohibited string would not start on a word boundary?

Reply:
```suggestion
      { no: "\\b[Nn]ote that", yes: "<nothing>" },
```

Reply:
There is a reason I didn't include it, but it may be lost to the mists of time. I'll see if git helps remind me.

Reply:
Git history is pretty short for the plugin. I can't find anything and adding `\\b` seems to not break any existing tests. So I'm going to add it to the plugin along with a test (using _denote that_ as text that should not be caught be a prohibited _note that_). So ignore the `\\b` suggestions here, but do let's get rid of the `\\s+` unless there's a known issue with getting rid of it.

Reply:
https://github.com/Trott/remark-lint-prohibited-strings/pull/9